bluebottle/recurring_donations/mails.py

```diff
@@ -19,16 +20,17 @@ def mail_monthly_donation_processed_notification(monthly_order):
 
     subject = _("Thank you for your monthly support")
 
-    translation.activate(cur_language)
+    translation.activate(receiver.primary_language)
```

Comment:
Now that we have a default primary language, you might as well remove the lines above this line that checks if there is a primary language.